content/sensu-core/2.0/reference/sensuctl.md

```diff
@@ -0,0 +1,185 @@
+  set-proxy-entity-id        set proxy entity id of a check
+  set-proxy-requests         set proxy requests for a check from file or stdin
+  set-publish                set publish of a check
+  set-runtime-assets         set runtime assets of a check
+  set-stdin                  set stdin of a check
+  set-subdue                 set subdue of a check from file or stdin
+  set-subscriptions          set subscriptions of a check
+  set-timeout                set timeout of a check
+  set-ttl                    set ttl of a check
+{{< /highlight >}}
+
+## Flags
+
+### Global
+Global flags modify settings specific to `sensuctl`, such as a Sensu host URL or
+[RBAC][1] information.
+
+{{< highlight shell >}}
+--api-url string        host URL of Sensu installation
+--cache-dir string      path to directory containing cache & temporary files 
+--config-dir string     path to directory containing configuration files
+--environment string    environment in which we perform actions (default "default")
+--organization string   organization in which we perform actions (default "default")
+{{< /highlight >}}
+
+### Local
+Local flags modify attributes on resources within Sensu. They differ per
+operation and per resource. Many, but not all, flags have shorthand flag
+equivalents. A list of flags can be found by running `sensuctl [TYPE] [command] -h`. 
+For example, the `check` command has the following local flags:
+
+{{< highlight line >}}
+  -c, --command string               the command the check should run
+      --cron string                  the cron schedule at which the check is run
+      --handlers string              comma separated list of handlers to invoke when check fails
+  -h, --help                         help for create
+      --high-flap-threshold string   flap detection high threshold (percent state change) for the check
+      --interactive                  Determines if CLI is in interactive mode
+  -i, --interval string              interval, in seconds, at which the check is run
+      --low-flap-threshold string    flap detection low threshold (percent state change) for the check
+      --proxy-entity-id string       the check proxy entity, used to create a proxy entity for an external resource
+  -p, --publish                      publish check requests (default true)
+  -r, --runtime-assets string        comma separated list of assets this check depends on
+      --stdin                        accept event data via STDIN
+  -s, --subscriptions string         comma separated list of topics check requests will be sent to
+  -t, --timeout string               timeout, in seconds, at which the check has to run
+      --ttl string                   time to live in seconds for which a check result is valid
+{{< /highlight >}}
+
+## Output
+
+sensuctl can be configured to return JSON instead of the default human-readable
+format.
+
+> sensuctl check info marketing-site --format json
```

Comment:
It looks like this format transforms the two consecutive hyphens `--` by an en dash `–` once generated by hugo, so when pasting and running that command:

```bash
$ sensuctl check info marketing-site –format json
[...]
Error: invalid argument(s) received
```

Reply:
Whoops, that to be wrapped in a highlight block!